site/content/3.12/data-science/arangographml/getting-started.md

```diff
@@ -372,23 +453,25 @@ arangoml.training.cancel_job(job_id)
 
 ### Model Selection
 
-Once the training is complete you can review the model statistics.
-The training completes 12 training jobs using grid search parameter optimization.
+Once the Training is complete you can review the model statistics.
+The Training Service returns **12 Models** using grid search parameter optimization.
  
-To select a model, use the projects API to gather all relevant models and choose
+To select a Model, use the Projects API to gather all relevant models and choose
 the one you prefer for the next step.
 
-The following examples uses the model with the highest validation accuracy,
+The following examples uses the model with the highest **test accuracy**,
 but there may be other factors that motivate you to choose another model.
+See the `model_statistics` field below for more information.
 
 ```py
-models = arangoml.projects.list_models(project_name=project_name, project_id=project_id, job_id=job_id)
+best_model = arangoml.get_best_model(
+    project.name,
+    training_job.job_id,
+    sort_parent_key="test",
+    sort_child_key="accuracy",
+)
 
-# Tip: Sort by accuracy
-models.models.sort(key=(lambda model : model.model_statistics["test"]["accuracy"]) , reverse=True)
-# The most accurate model is the first in the list
-model = models.models[0]
-print(model)
+print(best_model)
 ```
 
 **Expected output:**
```

Comment:
TODO: on the topic of Expected Outputs, I need to add/update expected output sections for all of our python snippets

Reply:
added in [41000e4](https://github.com/arangodb/docs-hugo/pull/423/commits/41000e4fa7afd500a81b7f7e9e9eeba3647af2db)